### PR TITLE
revamp docs home page

### PR DESCRIPTION
**Pull request by lewisjkl**
Here is what it looks like on desktop and mobile. Feel free to suggest changes to wording or whatever else.

screencapture-localhost-3000-smithy4s-2023-03-23-12_01_42
screencapture-localhost-3000-smithy4s-2023-03-23-12_01_26

### CODE REVIEW

File: modules/website/src/components/HomepageFeatures.js [hunk 1]
```diff
@@ -1,62 +1,65 @@
...
 
 const FeatureList = [
   {
-    title: 'Contract First',
-    Svg: require('../../static/img/undraw_sync_files.svg').default,
+    title: '1. Define API Contract',
+    lang: "smithy",
+    content: "service AdminService {\n  operations: [GetAllUsers]\n}",
     description: (
       <>
```
Comment (daddykotex) [inline modules/website/src/components/HomepageFeatures.js:12]: wow I love the code block `<>`

Comment (daddykotex) [reply modules/website/src/components/HomepageFeatures.js:12]: forget it, misunderstood that

File: modules/website/src/components/HomepageFeatures.js [hunk 2]
```diff
@@ -1,62 +1,65 @@
...
-        Smithy4s allows for idiomatic integration with your favourite
-        libraries/frameworks.
+        Smithy4s will use the Smithy model you define to generate Scala code including an interface that represents the service. This interface will contain one function per operation in the service.
       </>
     ),
   },
   {
-    title: 'Seamless API Dev',
-    Svg: require('../../static/img/undraw_add_files.svg').default,
+    title: '3. Create HttpRoutes',
```
Comment (kubukoz) [inline modules/website/src/components/HomepageFeatures.js]: Can we make this a bit more general and say "interpret into routes"?

File: modules/website/src/components/HomepageFeatures.js [hunk 3]
```diff
@@ -1,62 +1,65 @@
...
 import styles from './HomepageFeatures.module.css';
+import CodeBlock from '@theme/CodeBlock';
 
 const FeatureList = [
   {
-    title: 'Contract First',
-    Svg: require('../../static/img/undraw_sync_files.svg').default,
+    title: '1. Define API Contract',
+    lang: "smithy",
+    content: "service AdminService {\n  operations: [GetAllUsers]\n}",
```
Comment (Baccata) [inline modules/website/src/components/HomepageFeatures.js]: Do you think we could cram a little bit more in there ? Like : 

```smithy
operation GetUser {
   input := { 
     @required id: String 
   } 
   output := { 
     @required firstName: String 
     @required lastName: String 
   } 
} 

service AdminService { 
  operations: [GetUser] 
} 
```

Comment (lewisjkl) [reply modules/website/src/components/HomepageFeatures.js]: I'll give it a go and we can see how it looks

Comment (lewisjkl) [reply modules/website/src/components/HomepageFeatures.js]: screencapture-localhost-3000-smithy4s-2023-03-24-11_59_26

Comment (lewisjkl) [reply modules/website/src/components/HomepageFeatures.js]: Just pushed up with the changes. Feel free to merge if you like it we can revert back to the prior commit if not.

Comment (Baccata) [reply modules/website/src/components/HomepageFeatures.js]: Thanks. I've made some further changes, lmk what you think 

Screenshot 2023-03-27 at 15 21 19

Comment (lewisjkl) [reply modules/website/src/components/HomepageFeatures.js]: Looks great, I say send it! 🚀

**Pull request merged**